File: smash-ranks-android/app/src/main/java/com/garpr/android/misc/PreviousRankUtils.java
package com.garpr.android.misc;

import android.support.annotation.Nullable;

import com.garpr.android.models.Ranking;

public interface PreviousRankUtils {

    @Nullable
    Info checkRanking(@Nullable final Ranking ranking);


    enum Info {
        DECREASE, INCREASE
    }

}

File: smash-ranks-android/app/src/main/java/com/garpr/android/features/rankings/PreviousRankUtils.kt
package com.garpr.android.features.rankings

import com.garpr.android.data.models.RankedPlayer

interface PreviousRankUtils {

    fun getRankInfo(rankedPlayer: RankedPlayer?): Info?

    enum class Info {
        DECREASE, INCREASE, NO_CHANGE
    }

}
